admin/python/lsst/qserv/admin/configure.py

```diff
@@ -59,7 +59,7 @@
 ETC = 'etc'
 CLIENT = 'client'
 
-MYSQL = 'mysql'
+MARIADB = 'mariadb'
```

Comment:
What is the name of the process in mariadb? Is it mariadbd or still mysqld? If the latter, I'd definitely keep the mysql names, mariadb is a drop in replacement of mysql, isn't it? I worry otherwise we will never be sure whether to use "mysql" or "mariadb" name.